### PR TITLE
Install instructions

### PR DESCRIPTION
**Pull request by mangelajo**

### CODE REVIEW

**Comment by mangelajo** [comment]
@kirkbrauer I will merge this to continue work, but please feel free to comment / ask for changes , I will do follow ups, but I will make progress so we can interact with the website and see how it's looking by ourselves.

**Comment by kirkbrauer** [comment]
Make sense, thanks @mangelajo been super slammed this week ha

File: website/content/en/docs/Getting started/install.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+The direct helm install will auto-generate random router and controller secrets, but if you use ArgoCD make sure to set these values to unique values.
+{{% /alert %}}
+
+
+{{< tabpane text=true right=false >}}
+    {{% tab header="**Methods**:" disabled=true /%}}
+    {{< tab header="**Helm with OpenShift**" >}}
+
+        {{< highlight bash  >}}
+kind create cluster
```
Comment (mangelajo) [inline website/content/en/docs/Getting started/install.md:22]: remove kind line

File: website/content/en/docs/Getting started/install.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+            --set global.baseDomain=devel.jumpstarter.dev \
+            --set global.metrics.enabled=true \
+            --set jumpstarter-controller.grpc.mode=route \
+            --version=0.0.4-24-g1a8a159
+        {{< / highlight >}}
+    {{< /tab >}}
+        {{< tab header="**Helm with Kubernetes**" >}}
+
+        {{< highlight bash  >}}
+kind create cluster
```
Comment (mangelajo) [inline website/content/en/docs/Getting started/install.md:34]: remove kind here

**Pull request merged**